library/entropy_poll.c

```diff
@@ -115,6 +115,41 @@ static int getrandom_wrapper( void *buf, size_t buflen, unsigned int flags )
 #endif /* SYS_getrandom */
 #endif /* __linux__ || __midipix__ */
 
+/*
+ * Some BSD systems provide KERN_ARND.
+ * This is equivalent to reading from /dev/urandom, only it doesn't require an
+ * open file descriptor, and provides up to 256 bytes per call (basically the
+ * same as getentropy(), but with a longer history).
+ *
+ * Documentation: https://netbsd.gw.com/cgi-bin/man-cgi?sysctl+7
+ */
+#if (defined(__FreeBSD__) || defined(__NetBSD__)) && !defined(HAVE_GETRANDOM)
+#include <sys/param.h>
+#include <sys/sysctl.h>
+#if defined(KERN_ARND)
+#define HAVE_SYSCTL_ARND
+
+static int sysctl_wrapper ( void *buf, size_t buflen )
```

Comment:
Style: `sysctl_wrapper(` (no space before the opening parenthesis). The space breaks `tests/scripts/recursion.pl`, which makes some rather strong assumptions about coding style in order to locate functions using regexes.

Reply:
Naming nitpick: I think `sysctl_arnd_wrapper()` would be a bit better (this isn't a generic sysctl wrapper).

Reply:
Thanks for the feedback! I was very confused why recursion.pl was failing.